src/main/java/com/bugsnag/android/Bugsnag.java

```diff
@@ -24,6 +25,7 @@ private Bugsnag() {}
      */
     public static Client init(Context androidContext) {
         client = new Client(androidContext);
+        configureClientObservers();
```

Comment:
This can't go in the constructor of the client object because until the end of the constructor there is no client object to add observers to. I think there needs to be another call added if the client object is being used without the Bugsnag class